core/color.cpp

```diff
@@ -122,7 +122,7 @@ float Color::get_h() const {
 
 	float delta = max - min;
 
-	if (delta == 0)
+	if (Math::is_zero_approx(delta))
```

Comment:
I dont kinda like doing this. Anything should be good as long as it's not zero.

Reply:
Good point. I have changed all of the code that was `== 0` back to `== 0`.